lib/ibanity/client.rb

```diff
@@ -25,6 +25,7 @@ def initialize(
       @ponto_connect_client_secret  = ponto_connect_client_secret
       @certificate                  = OpenSSL::X509::Certificate.new(certificate)
       @key                          = OpenSSL::PKey::RSA.new(key, key_passphrase)
+      @http_debug                   = !!ENV["IBANITY_HTTP_CLIENT_DEBUG"]
```

Comment:
We should add this to the README. Maybe it should be (an optional) part of the client initialization instead of an environment variable?

Reply:
done